### ISSUE TITLE
[Feature Request] Allow setup some Settings while Phone get flashed

### ISSUE THREAD

**Issue by Mannshoch** [ubports/ubports-installer#57]
While UBports is installing It may could allow users to preset the System.
![grafik](https://user-images.githubusercontent.com/7795891/33948923-72bbc22c-e028-11e7-8c35-e0433d79860d.png)

**Comment by NeoTheThird** [comment]
Yeah, thought about something like that for a while. It would have to work really well, though, and at the moment there's no interface on the phone that will accept the instructions you send and turn them into settings... The only way that would work is, if the data was sent to the phone after the installation and first boot is complete, and i don't know if people would keep their phone connected that long, that could also cause a lot of confusion and frustration.

It might make more sense to use the time during installation to display some information about the os (similar to the ubiquity installer or the ubuntu mate welcome menue) and leave all settings to the first-setup-wizard on the device.

**Comment by Mannshoch** [comment]
I would further Vote for a solution for this. I'm sure that you get a strategy to be able to set-up some settings from Computer. Maybe you could split that into a separate Part of the application so I'm able to setup some settings every time I connect my Phone to my computer but be also able to flash a system if needed. I have no problem to setup things on phone but a Mouse and Keyboard is even nicer to use. A big plus is also that I do not need any thing saved on desktop PC (except maybe an Import/export of settings) and could use it whenever I need it.

**Comment by NeoTheThird** [comment]
Our [recovery](https://github.com/ubports/android_bootable_recovery/blob/ubp-5.1/system-image-upgrader#L300) has some options we could pass to [systemimage](https://github.com/ubports/system-image-node-module/blob/master/src/client.js#L185).
